scripts/upload_asset.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Assure parameters are correct.
+if [ $# -lt 2 ]; then
+    echo "Usage: upload_asset.sh <FILE> <TOKEN>"
+    exit 1
+fi
+
+repo="chrisduerr/alacritty"
+file_path=$1
+bearer=$2
+
+echo "Starting asset upload from $file_path to $repo."
```

Comment:
```suggestion
printf "Starting asset upload from $file_path to $repo.\n"
```

Reply:
I have no interest in using only echo in this script.

Reply:
I don't really like mixing 2 things.

Reply:
I do though. There's nothing wrong with mixing these two for the things they're better at. Using printf just to not use echo when echo is just simpler makes little sense.